app/services/csv/lettings_log_csv_service.rb

```diff
@@ -174,6 +178,8 @@ def value(attribute, log)
         when "labels"
           PERSON_DETAILS.find { |key, _value| key == attribute }[1]["refused_label"]
         end
+      elsif NOT_IMPLEMENTED_FIELDS.include? attribute
+        nil
```

Comment:
Do we need this? Wouldn't else return nil (assuming we don't have these values set)